scripts/jtsync/README.md

```diff
@@ -0,0 +1,58 @@
+# jtsync - synchronize status from jenkins to trello
+    Usage: jtsync --ci SERVICE (--matrix|--job) JOB_STATUS
+            --ci SERVICE                 Which ci is used (suse or opensuse)
+            --matrix NAME,PROJECT        Set status of a matrix job
+            --job NAME                   Set status of a normal job
+
+Matrix job cards are selected in this form:
+
+    $ jtsync --ci suse --matrix cloud-trackupstream,Devel:Cloud:7:Staging
+
+The script never returns anything other than `0` to make sure jenkins runs are not interruped
+by the script.
+
+
+## Run the script on a developer machine
+Install `bundler` before continue:
+
+    cd <path to jtsync directory>
+    bundler install --path=~/.bundle-jtsync
+    bundle exec jtsync.rb
+    ...
+
+## How to test:
+In general there is the `test_jtsync` script to run an manual test run. You need
+to create credentials for the trello API (you can create credentials [here](https://trello.com/app-key))
+Place them in your local `.netrc` file:
+
+    machine api.trello.com
+      login <developer token>
+      password <member token>
+
+After that create a new trello board or clone [https://trello.com/b/fL7fv67z/jtsync](https://trello.com/b/fL7fv67z/jtsync).
```

Comment:
Do we really need each developer to create their own Trello board for testing?  That seems pretty wasteful.  Can't we just reuse the existing test board?  That would also remove the need to edit the hardcoded board id and ensure that the correct board structure exists.  It seems unlikely that we'd have multiple developers testing different changes to jtsync at the same time - and if that happened, those developers should probably be collaborating anyway.